kuka_sunrise_control/teleop_guided_robot/package.xml

```diff
@@ -3,9 +3,9 @@
 <package format="3">
   <name>teleop_guided_robot</name>
   <version>0.0.0</version>
-  <description>TODO: Package description</description>
+  <description>Package for managing the connection between ROS and KUKA Sunrise FRI</description>
```

Comment:
But this is not the purpose of the teleop_guided_robot package, is it?

Reply:
True, I will update this
But I think we should consider moving the system_manager to another package, as it has a more general purpose than just starting the teleop_guided motion